### ISSUE TITLE
What has to be done to use this with openGL

### ISSUE THREAD

**Issue by pseregiet** [tommai78101/citro3d-physics#1]
Hi. I've got here from the cu3e project. I myself thought of porting cu3e to C but it seems you have done it already. The readme says this is an extension to the citro3D library for 3DS so I am wondering how much effort would it be to make this library standalone assuming we have a C math library (I use HomeMadeMath in my C code). I could try making this library standalone but I thought I'd ask you about some pointers on what would have to be done since you're more familiar with the code.

**Comment by tommai78101** [comment]
You will want to grab the citro3D math libraries and other common libraries that this project references. Things like the stdint, the define macros, and so on. Also, make sure you don't reference anything that is exclusive to the citro3D and the Nintendo 3DS class objects, I believed, that I added in those references in the Pica engine.

If I were you, I would start out cu3e from scratch and make it a general-purpose C library, something that isn't like my engine which has a hard dependency on the Nintendo 3DS homebrew and graphics libraries. You are free to use some neat C tricks I wrote in the Pica engine, that involves me porting C++ features to C.

I gotta say, there is a part of the code where I'm stuck (stuck to this day) trying to make it work on the 3DS. And because at the time of writing the project, there weren't any ways of debugging it on a hardware and testing it with homebrew libraries, I felt like I was porting cu3e blind. Thus, you will have to figure out yourself if the ported C code you write is really working as intended, or not.
